### ISSUE TITLE
Save as HTML raises an error

### ISSUE THREAD

**Issue by gquintana** [asciidoctor/atom-asciidoc-preview#224]
## Description

On the Asciidoc preview, right click then Saves as HTML...

```
Error: Exited with code 1
    at ChildProcess.<anonymous> (C:\Java\atom\.atom\packages\asciidoc-preview\node_modules\opn\index.js:83:13)
    at ChildProcess.g (events.js:286:16)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```
Despite the error message, the HTML document is properly saved.

- Atom version: 1.15.0 ia32
- OS: Windows 7
- `asciidoc-preview` version: 2.7.2
- `language-asciidoc` version: 1.7.0

## Screenshots
![image](https://cloud.githubusercontent.com/assets/755587/24195390/cc29fb22-0ef9-11e7-8b79-e2379baf1ff4.png)

**Comment by ldez** [comment]
Can you update to 2.9.0 ?

**Comment by ldez** [comment]
fix in 2.9.1 🎉
